docs/new-architecture-app-modules-android.md

```diff
@@ -272,8 +363,78 @@ public class MyApplication extends Application implements ReactApplication {
                 return new MyApplicationTurboModuleManagerDelegate.Builder();
             }
         };
+}
+```
+
+</TabItem>
+<TabItem value="kotlin">
+
+```kotlin
+class MyApplication() : Application(), ReactApplication {
+    private val mReactNativeHost: ReactNativeHost =
+        object : ReactNativeHost(this) {
+            /* ... */
+            val useDeveloperSupport: Boolean
+                get() {
+                    /* ... */
+                }
+
+            val packages: List<Any>
+                protected get() {
+                    val packages: MutableList<ReactPackage> = PackageList(this).getPackages()
+
+                    // Add those lines
+                    packages.add(
+                        object : TurboReactPackage() {
+                            @Nullable
+                            fun getModule(
+                                name: String,
+                                reactContext: ReactApplicationContext?
+                            ): NativeModule? {
+                                if ((name == NativeAwesomeManager.NAME)) {
+                                    return NativeAwesomeManager(reactContext)
+                                } else {
+                                    return null
+                                }
+                            }
```

Comment:
```suggestion
                            ): NativeModule? =
                                if ((name == NativeAwesomeManager.NAME)) {
                                    NativeAwesomeManager(reactContext)
                                } else {
                                    null
                                }
                            
```

Reply:
Fixed, Tested the change for compilation in [RNNewArchitectureApp](https://github.com/react-native-community/RNNewArchitectureApp) using npx react-native run-android command